docs/eventing/getting-started.md

```diff
@@ -33,22 +40,27 @@ kubectl get pods --namespace knative-eventing
 If the `knative-eventing` namespace or the `imc-controller-*` does not exist, use the following steps to install Knative Eventing with the in-memory channel:
 
 1. Make sure that you have a functioning Kubernetes cluster. See the [Comprehensive Install guide](../install) for more information.
+
    - Old versions of Knative Serving doesn't necessarily work well with latest Knative Eventing, so try to install the latest version of Knative Serving.
+
    - If your Kubernetes cluster comes with pre-installed Istio, make sure it has `cluster-local-gateway` [deployed](https://github.com/knative/serving/blob/master/DEVELOPMENT.md#deploy-istio). Depending on which Istio version you have, you'd need to apply the `istio-knative-extras.yaml` in the corresponding version folder at [here](https://github.com/knative/serving/tree/{{< branch >}}/third_party).
-2. Install the Eventing CRDs by running the following command:
+
+1. Install the Eventing CRDs by running the following command:
 
     ```sh
     kubectl apply --selector knative.dev/crd-install=true \
     --filename https://github.com/knative/eventing/releases/download/{{< version >}}/release.yaml
     ```
 
-3. Install the Eventing sources by running the following command:
+1. Finish installing Eventing resources by running the following command:
 
     ```sh
     kubectl apply --filename https://github.com/knative/eventing/releases/download/{{< version >}}/release.yaml
     ```
 
-4. Confirm that Knative Eventing is correctly installed by running the following command:
+   Installing the CRDs first can prevent race conditions and failures during the install.
```

Comment:
Would it make sense to describe briefly what race conditions are, or link to somewhere?

Reply:
Done! I couldn't find anything to link to, but added some more info.

Reply:
Thanks!